css/src/components/notification.scss

```diff
@@ -0,0 +1,101 @@
+$notification-background: $secondary-background !default;
+$notification-padding: $spacer-5 !default;
+
+$notification-font-size: $font-size-7 !default;
+$notification-color: $text !default;
+
+$notification-border-color: $border-dark !default;
+$notification-border-width: $border-width !default;
+$notification-border-radius: $border-radius !default;
+
+.notification {
+	display: block;
+	position: relative;
+	padding: $notification-padding;
+	border: $notification-border-width solid $notification-border-color;
+	border-radius: $notification-border-radius;
+	outline-color: $text;
+	background-color: $notification-background;
+	color: $notification-color;
+	font-size: $notification-font-size;
+	word-wrap: break-word;
+	word-break: break-word;
+
+	&.is-loading {
+		color: transparent !important;
+
+		> :first-child {
+			margin-inline-start: calc($notification-padding + 0.375em);
+		}
+
+		&::before {
+			@include loader;
+
+			position: absolute !important;
+			inset-block-start: $notification-padding;
+			inset-inline-start: $notification-padding;
+			border-color: transparent transparent $notification-color $notification-color !important;
+		}
+	}
+
+	@each $name, $color-set in $colors {
+		$base: nth($color-set, $color-index-base);
+		$dark: nth($color-set, $color-index-dark);
+		$background: nth($color-set, $color-index-background);
+
+		&.notification-#{$name} {
+			border-color: $base;
+			background-color: $background;
+
+			@if $name == 'info' or $name == 'primary' {
+				color: $dark;
+			} @else {
+				color: $base;
+			}
+
+			&.is-loading::before {
+				border-color: transparent transparent $dark $dark !important;
+			}
+
+			/* stylelint-disable max-nesting-depth */
+
+			.button-close {
+				color: inherit;
+
+				&:hover,
+				&.is-hovered {
+					color: $dark;
+				}
+			}
+
+			/* stylelint-enable max-nesting-depth */
+		}
+	}
+
+	.notification-title,
+	a:not(.button) {
+		color: currentColor;
+		font-weight: $weight-semibold;
+	}
+
+	.notification-title {
```

Comment:
title seems too close to following text for me. Margin block end? <img width="613" alt="image" src="https://user-images.githubusercontent.com/30843002/230229628-c61c3e37-eddb-4314-a8ad-bab574237ba9.png">

Reply:
Updated, does it look better?

![image](https://user-images.githubusercontent.com/57374379/230495755-ad451e0e-44c2-436e-867d-754b5284134d.png)